package.json

```diff
@@ -26,6 +26,7 @@
     "@babel/preset-typescript": "7.1.0",
     "@commitlint/cli": "7.5.0",
     "@commitlint/config-conventional": "7.5.0",
+    "@types/firefox-webext-browser": "^65.0.0",
```

Comment:
can lock dependency version instead? let renovate control the version
it can be done by `npm i -DE @types/firefox-webext-browser`